.github/workflows/reusable-sync-changes.yaml

```diff
@@ -1,4 +1,4 @@
-name: Clone branch
+name: Sync changes
```

Comment:
I think we should be more specific here what's getting synced, otherwise it could get confusing in the individual repos. 
E.g. `Sync changes from EE to CE repository`, so that everyone knows directly what went wrong in case an Action fails.